src/main.ts

```diff
@@ -110,7 +110,8 @@ const initComponents = () => {
 	// Inject modal dialog.
 	const parser = new DOMParser();
 	const dialog = parser.parseFromString(content, 'text/html');
-	const modalContents = document.body.appendChild(dialog.body);
+	const firstDiv = dialog.body.querySelector('div') || document.createElement('div');
```

Comment:
I see what you are doing here, but this would not account for either multiple elements in the HTML, or a change to the elements types themselves. (.e.g, a change from a `<div>` to a `<dialog>`). 

I think the best course of action is to insert a dummy div and then append the children of body to that. I can take a stab at it later today.